docs/pages/database-access/guides/rds.mdx

```diff
@@ -135,12 +135,59 @@ Teleport database agent will be using (user or role).
   }
   ```
   </TabItem>
+  <TabItem label="RDS proxy / IAM user">
```

Comment:
With these two new `TabItem`s, this `Tabs` component might be a bit overstuffed. I think we should either:
- Leave this as it is for now and wait for the result of [this issue](https://github.com/gravitational/next/issues/796), i.e., for an alternative component to become available
- Use H4-level subheadings instead of namespaced tab labels. For example, rather than "RDS / IAM user" and "RDS / IAM role" labels, we could have an H4 heading that says, `#### RDS` then include a `Tabs` component with `TabItem`s for "IAM user" and "IAM role." Same for `#### Aurora` and `#### RDS proxy`
- The previous option, except using H4 headings for "IAM user" and "IAM role," each with a `Tabs` component that includes labels for "RDS, " "Aurora," and "RDS proxy"

Reply:
I tried option 2 with H4. It took quite some scrolling to view all H4s and it feels very easy to loose track which H3 we are actually on. 

What's your option on nested tabs? I quickly tried it and i feel it looks ok. If not, i can wait for the issue you linked above.

Reply:
I think it should be okay to leave this as it is, then—we can wait until [this issue](https://github.com/gravitational/next/issues/796) gets resolved and rework the high-count `Tabs` components to use the newly designed alternative!